sql-native/src/main/java/org/ballerinalang/sql/nativeimpl/ExecuteProcessor.java

```diff
@@ -104,6 +110,11 @@ public static Object nativeExecute(BObject client, Object paramSQLString) {
     }
 
     public static Object nativeBatchExecute(BObject client, BArray paramSQLStrings) {
+        return nativeBatchExecute(client, paramSQLStrings, StatementParameterProcessor.getInstance());    
+    }
+
+    public static Object nativeBatchExecute(BObject client, BArray paramSQLStrings,
```

Comment:
Same as above, logic can be moved to the calling method.

Reply:
+1

Reply:
Resolved with [`666f5cm`](https://github.com/ballerina-platform/module-ballerina-sql/pull/104/commits/666f5c49ee9e3cb4dc0575a792bc643ee9523451)